tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -755,29 +755,29 @@ def test_send_status_to_db(self, cluster):
 
     @pytest.mark.level("minimal")
     @pytest.mark.clustertest
-    def test_status_scheduler_basic_flow(self, cluster):
+    def test_status_scheduler_basic_flow(self, ondemand_aws_https_cluster_with_auth):
+        cluster = ondemand_aws_https_cluster_with_auth
+        cluster.restart_server()
 
         if not cluster.config().get("resource_subtype") == "OnDemandCluster":
             pytest.skip(
                 "This test checking pinging cluster status to den, this could be done only on OnDemand clusters."
             )
 
+        cluster_uri = rh.globals.rns_client.format_rns_address(cluster.rns_address)
```

Comment:
maybe im missing something but what's the core change here?

Reply:
to make sure we could assert that `'HTTP Request: POST {api_server_url}/resource/{cluster_uri}/cluster/status "HTTP/1.1 200 OK"' in cluster_logs` (line 774)

prob will change soon tho, if we'll print this log in a debug mode.